test/govspeak_attachment_link_test.rb

```diff
@@ -22,4 +22,15 @@ def render_govspeak(govspeak, attachments = [])
     assert_match(/<span class="gem-c-attachment-link">/, rendered)
     assert_match(%r{href="http://example.com/attachment.pdf"}, rendered)
   end
+
+  test "renders an attachment link inside a paragraph" do
```

Comment:
I'm wondering if we should have a couple more tests here:

   - The edge case of not inserting a paragraph when an attachment link is not found
   - A more specific test (possibly the same new one) to test the link and subsequent words are put in the same paragraph element

Reply:
We've got that first test you suggest: https://github.com/alphagov/govspeak/blob/d3b2914b1151a64d0b08a44b556b4b0fcf44b7be/test/govspeak_attachment_link_test.rb#L10-L12

Other test makes sense but it's a bit grim to do. I've had a go at it with nokogiri and pushed it up.

Reply:
Oh yes - sorry for missing that one.